components/ai/code/ai-bbe/natural_programming_vs_code_features.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Natural Programming: Bridge Requirements and Code"
+description: "Leverage Generative AI to keep requirements, code, and documentation in sync.
+
+
+- **AI-powered code generation** – Generate code from high-level business requirements.
```

Comment:
Since everything is AI-powered and we mention Gen AI in L3, shall we remove "AI-powered" from code and test gen, if not, we have to introduce to all

```suggestion
- **Code generation** – Generate code from high-level business requirements.
```